src/js/utils/option-utils.jsx

```diff
@@ -81,7 +81,7 @@ export const debounceLocationsFetch = (
 export const debounceGlobalSearch = (waitTime, minSearchLength) =>
   _.debounce((searchTerm, callback) => {
     if (searchTerm && searchTerm.length >= minSearchLength) {
-      apiClient.get(`/json/globalSearch?term=${searchTerm}`)
+      apiClient.get(encodeURI(`/json/globalSearch?term=${searchTerm}`))
```

Comment:
@alannadolny How this will look from the backend point of view? Test it out on some other special characters, perhaps some test out some French product names.